src/gui/DatabaseSettingsWidget.cpp

```diff
@@ -168,6 +171,8 @@ void DatabaseSettingsWidget::save()
         }
     }
 
```

Comment:
I know this is a simple implementation of a radio button set, but it's not scalable and we really should be wrapping the buttons in a QButtonGroup and calling checkedButton()

Reply:
There is no QButtonGroup widget in the designer. I could add a generic QWidget and promote it, but I don't really see the benefit. Button groups work with any parent widget and all the QButtonGroup is giving us is the index of the selected button, but not a way to set the active button, so it's not really making things easier. The documentation for QButtonGroup also suggests using a QGroupBox as a container with a visual representation.

Reply:
I was surprised to see the QGroupBox did not inherit from QButtonGroup...

Reply:
As I said, exclusive buttons are not specific to QButtonGroup.